projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -28,6 +28,24 @@ public static function init() {
 			return;
 		}
 
+		add_filter(
+			'load_script_textdomain_relative_path',
+			function ( $relative ) {
+				// Check if $relative is a string and contains the required segment
+				if ( str_contains( $relative, 'vendor/automattic/jetpack-mu-wpcom/' ) ) {
+					$pattern = '#^(?:production|staging|moon|sun)/#'; // Matches the environment prefix on Simple Sites
+
+					if ( preg_match( $pattern, $relative ) ) {
+						$relative = preg_replace( $pattern, '', $relative ); // Remove the environment prefix
+					}
```

Comment:
Could this be simplified to
```suggestion
					$relative = preg_replace( $pattern, '', $relative ); // Remove the environment prefix
```
?